exporter/chronicleforwarderexporter/README.md

```diff
@@ -0,0 +1,55 @@
+# Chronicle Forwarder Exporter
```

Comment:
Make sure this is linked to in `docs/exporters.md`

Reply:
linked